clusterloader2/testing/load/config.yaml

```diff
@@ -0,0 +1,147 @@
+automanagedNamespaces: 3
+tuningSets:
+- name: Uniform100qps
+  qpsLoad:
+    qps: 100
+- name: Uniform5qps
+  qpsLoad:
+    qps: 5
+steps:
+- measurements:
+  - Identifier: GlobalTimer
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: CreateSaturationPodsTimer
+    Method: Timer
+    Params:
+      action: start
+  - Identifier: APICallLatency
+    Method: APICallLatency
+    Params:
+      action: reset
+# Create RCs
+- phases:
+  - namespaceRange:
+      min: 1
+      max: 3
+    replicasPerNamespace: 1
+    tuningSet: Uniform100qps
+    objectBundle:
+    - basename: saturation-service
+      objectTemplatePath: service.yaml
+    - basename: saturation-rc
+      objectTemplatePath: rc.yaml
+      templateFillMap:
+        ReplicasMin: 30
+        ReplicasMax: 30
+- measurements:
+  - Identifier: WaitForRunningSaturationRCs
```

Comment:
-> WaitForRCsRunning

Reply:
`Saturation` phrase removed from the config.